packages/suite/src/components/suite/modals/AddAccount/components/MoreCoins.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { useState } from 'react';
+import { AnimatePresence, motion } from 'framer-motion';
+import styled, { css } from 'styled-components';
+import { Icon, P } from '@trezor/components';
+import { Translation } from '@suite-components';
+import { useTheme } from '@suite-hooks';
+
+const animationDuration = 0.24;
```

Comment:
Why exactly 0.24? We use 0.2, 0.3, 0.4, 0.5, 0.6 and 1s in the codebase.

Reply:
I copied it from here: https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/config/suite/animation.ts 🤷 

As I mentioned somewhere, I don't like this animation, especially because of the simultaneous animation of height and width, I would like to spend more time on making it nice, but not sure if it's worth it (in terms of priorities)